docs/generated/sql/diagrams/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package diagrams
+
+//go:generate go run ../../../../pkg/cmd/docgen/main.go ../../../../pkg/cmd/docgen/funcs.go ../../../../pkg/cmd/docgen/diagrams.go grammar --base ./diagrams --railroad ../../../../build/Railroad.jar --addr ../../../../pkg/sql/parser/sql.y
```

Comment:
We should consider `go install`ing docgen and letting the Makefile keep it up to date. (In a future PR.) Go makes this so damn gross.

Reply:
yeah, i just didn't want to force re-building it whenever we built tools, since it does depend on e.g. sql/parser, since not everyone actually needs to run it -- only those making sql lang changes do. I guess we make everyone build protoc even if they don't make proto changes though, so eh. I wish we had better just-in-time tool binary mgmt.

Reply:
MORE DOTS